components/resources/wallet_strings.grdp

```diff
@@ -427,6 +427,8 @@
   <message name="IDS_BRAVE_WALLET_TRANSACTION_STATUS_ERROR" desc="Status label for error transaction">Error</message>
   <message name="IDS_BRAVE_WALLET_TRANSACTION_STATUS_DROPPED" desc="Status label for dropped transaction">Dropped</message>
   <message name="IDS_BRAVE_WALLET_TRANSACTION_STATUS_SIGNED" desc="Status label for signed transaction">Signed</message>
+  <message name="IDS_BRAVE_WALLET_ON" desc="Transaction details transacted on label">On</message>
```

Comment:
The translators will want to know the context here. What is this label referring to?

Reply:
Added some context in the `desc`